### PR TITLE
[WIP] Win10 only

### PR DESCRIPTION
**Pull request by janpio**
### What does this PR do?

Removes all logic for non-Win10.

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Comment by brodybits** [comment]
I feel a bit ambivalent about this change right now. I continue to support legacy sqlite plugin version for pre-win10, not sure that the need has completely gone away yet.

**Comment by mobidev111** [comment]
Windows 8 / Windows 8.1 is gone - about 7% marketshare [source](https://netmarketshare.com/operating-system-market-share.aspx?options=%7B%22filter%22%3A%7B%22%24and%22%3A%5B%7B%22deviceType%22%3A%7B%22%24in%22%3A%5B%22Desktop%2Flaptop%22%5D%7D%7D%5D%7D%2C%22dateLabel%22%3A%22Trend%22%2C%22attributes%22%3A%22share%22%2C%22group%22%3A%22platformVersion%22%2C%22sort%22%3A%7B%22share%22%3A-1%7D%2C%22id%22%3A%22platformsDesktopVersions%22%2C%22dateInterval%22%3A%22Monthly%22%2C%22dateStart%22%3A%222017-06%22%2C%22dateEnd%22%3A%222018-05%22%2C%22plotKeys%22%3A%5B%7B%22platformVersion%22%3A%22Windows%208%22%7D%2C%7B%22platformVersion%22%3A%22Windows%208.1%22%7D%5D%2C%22segments%22%3A%22-1000%22%7D).

So at least for cordova development activities this shouldn't be a priority.

If someone still relies on Windows 8/8.1 without a realistic upgrade path to 10.0 within the next year: please speak up now.

**Comment by janpio** [comment]
If someone still uses Windows 8/8.1 they can just continue using the current release that still supports it. I'm not even sure if VS17 really still supports 8/8.1 anyway...

**Comment by brodybits** [comment]
@janpio should we keep this PR open?

**Comment by brodybits** [comment]
What is the status of this proposal?

**Comment by janpio** [comment]
- WIP in title, so unfinished
- Conflicting files, so would need additional work to get mergable
- Tests are failing because no idea why

File: cordova-js-src/splashscreen.js [hunk 1]
```diff
@@ -19,15 +19,14 @@
  *
 */
 
-var isWp81 = navigator.appVersion.indexOf("Windows Phone 8.1") !== -1;
 var isWp10 = navigator.appVersion.indexOf("Windows Phone 10") !== -1;
-var isPhoneDevice = isWp81 || isWp10;
+var isPhoneDevice = isWp10;
```
Comment (brodybits) [inline cordova-js-src/splashscreen.js]: I would vote to drop `isWp*` and `isPhone*`, we just check if we are dealing with UWP, appx-web, etc.

Comment (janpio) [reply cordova-js-src/splashscreen.js]: Fell free to do that in a PR, but don't take this as the base as it really was just a PoC to see what happens when I delete some code.

**Comment by prasadpandari** [comment]
@janpio 
Did you had chance to do any further work. I have similar requirement.

**Pull request closed without merging**